### PR TITLE
Adds HttpHeadersUtils.

### PR DESCRIPTION
**Pull request by pennello**
https://opentable.atlassian.net/browse/OTPL-1774

### CODE REVIEW

File: shared/src/main/java/com/opentable/jaxrs/util/ContainerRequestContextHttpHeaders.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.core.Cookie;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.MultivaluedMap;
+
+/**
+ * Implementation of {@link HttpHeaders} based on {@link ContainerRequestContext}.
+ */
+public class ContainerRequestContextHttpHeaders implements HttpHeaders {
```
Comment (stevenschlansker) [inline shared/src/main/java/com/opentable/jaxrs/util/ContainerRequestContextHttpHeaders.java]: Doesn't need to be public

Comment (pennello) [reply shared/src/main/java/com/opentable/jaxrs/util/ContainerRequestContextHttpHeaders.java]: Good idea.  Implemented in ad462024875ce8c197ff77d58ecfce1af5caeb0c.

**Comment by pennello** [comment]
Thanks Steven!

**Comment by pennello** [comment]
fyi @strykerd @mikebell90 @rushalias

**Pull request merged**